src/Word.java

```diff
@@ -0,0 +1,7 @@
+import java.util.Random;
+
+public interface Word {
```

Comment:
Also name doesn't match with what interface is trying to do. Maybe `WordsSource` would be better?

Reply:
you're right